docs/elements/hx-menuitem-group/index.html

```diff
@@ -0,0 +1,15 @@
+---
+title: <hx-menuitem-group>
+---
+{% extends 'element.njk' %}
+{% block content %}
+
+<section>
+  <p>
+    An element that provides grouping of individual <a href="elements/hx-menuitem">
+    &lt;hx-menuitem&gt;</a> elements within an <a href="elements/hx-menu">&lt;hx-menu&gt;</a>. 
```

Comment:
Nit: As this isn't a sentence, I wouldn't use a period (unless you're doing it across the site, in which case consistency is better).

Reply:
Actually, I'd rather see it rewritten:
```html
A <code>&lt;hx-menuitem-group&gt;</code> is an element that provides grouping of individual <a href="elements/hx-menuitem">&lt;hx-menuitem&gt;</a> elements within an <a href="elements/hx-menu">&lt;hx-menu&gt;</a>. 
```

Reply:
yeah i was wondering this -- we are using periods across the site but may want to look into that!